src/Appwrite/Platform/Workers/Hamster.php

```diff
@@ -368,6 +367,21 @@ private function getStatsForOrganization(Document $organization, Database $dbFor
                 throw new \Exception('Membership not found. Skipping organization : ' . $organization->getId());
             }
 
+            $billingPlan = $membership->getAttribute('billingPlan', null);
+            $billingPlanDowngrade = $membership->getAttribute('billingPlanDowngrade', null);
```

Comment:
This is the membership collection.. membership collection does not have these attributes. these attributes are part of the teams collection

Reply:
Yeah, it should be $organization.
fixed